examples/gce/.github/generate-reports.sh

```diff
@@ -0,0 +1,61 @@
+#!/bin/bash
+
+# Copyright 2020 The Kubernetes Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+set -o errexit
+set -o nounset
+set -o pipefail
+
+echo "Extracting reports from sonobuoy"
+SONOBUOY_REPORTS=/tmp/reports
+
+INGRESS_CONTROLLER=${INGRESS_CONTROLLER:-'N/A'}
+CONTROLLER_VERSION=${CONTROLLER_VERSION:-'N/A'}
+
+TEMP_CONTENT=$(mktemp -d)
+
+docker run \
+  -e BUILD="$(date -u +'%Y%m%d%H%M')" \
```

Comment:
can we put T and : in the timestamp so it's parseable by other tools 

"%Y-%M-%dT%H:%M"

Reply:
done